package.json

```diff
@@ -1,17 +1,20 @@
 {
-  "name": "Tanki",
+  "name": "tanki",
```

Comment:
Oops, my bad: the name must follow [some rules](https://docs.npmjs.com/files/package.json#name).